### PR TITLE
feat(core): describe initAction

### PR DESCRIPTION
**Pull request by after-finitude**

### CODE REVIEW

File: docs/glossary.md [hunk 1]
```diff
@@ -92,6 +92,16 @@ const countDoubledAtom = declareAtom(0, on => [
 // const countDoubledAtom = map(count, count => count * 2)
 ```
 
+### Atom init
```
Comment (artalar) [inline docs/glossary.md:95]:
```suggestion
### Dynamic initial state
```

File: docs/glossary.md [hunk 2]
```diff
@@ -92,6 +92,16 @@ const countDoubledAtom = declareAtom(0, on => [
...
+
+If you want to init atom you can subscribe on init action
```
Comment (artalar) [inline docs/glossary.md:97]:
```suggestion
If you need to dynamically generate atom state (on subscription) you can derive it from `init` action

> See more about atom state [lifecycle](https://reatom.js.org/#/glossary?id=states-laziness)
```

File: package.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
     "test": "lerna run test",
     "size": "lerna run size",
     "lint": "eslint --ext .ts,.tsx,.js,.jsx packages/",
-    "prettify": "prettier --write '**/*.{js,ts,md}' !**/babel-plugin/test/**",
+    "prettify": "cross-env prettier --write '**/*.{js,ts,md}' !**/babel-plugin/test/**",
```
Comment (artalar) [inline package.json:12]: Remove it, please. It must be placed under other PR

File: docs/glossary.md (hunk 1 above)
Comment (after-finitude) [reply docs/glossary.md]: Done

File: docs/glossary.md (hunk 2 above)
Comment (after-finitude) [reply docs/glossary.md]: Done

File: package.json (hunk 3 above)
Comment (after-finitude) [reply package.json]: Done

**Comment by artalar** [comment]
@allcontributors please add @after-finitude for code

**Pull request merged**